imgix/luminous
# fix: fix arrow navigation

## Description

This PR fixes a regression in a previous release of this library, where arrow navigation was broken.

The problem was that when the Closure compiler was integrated into this library, the way the settings were passed around the objects was broken as Closure rewrites object key names. This fixes that.

![kapture 2018-11-17 at 16 01 52](https://user-images.githubusercontent.com/615334/48662419-21e36980-ea82-11e8-9503-7d8c369b7314.gif)

Fixes #74 

### Bug Fix

- [N/A] Add new passing unit tests to cover the code introduced by your PR
- [N/A] Update the readme
- [N/A] Update or add any necessary API documentation



## Steps to Test

Check that tests still pass.

To test locally:

1. Clone PR and `npm install && npm run build`
2. Run a webserver and open `index.html` in a browser
3. Click on an image at the bottom
4. Check that arrow navigation works
